circle.yml

```diff
@@ -41,6 +41,41 @@ commands:
               curl -L https://github.com/WebAssembly/wabt/releases/download/1.0.19/wabt-1.0.19-$os.tar.gz | sudo tar xz --strip 1
             fi
 
+  install_testfloat:
+    description: "Install TestFloat test suite tools"
+    steps:
+      - restore_cache:
+          name: "Get TestFloat from cache"
+          key: &testfloat-cache-key TestFloat-3e-{{arch}}-v2
+      - run:
+          name: "Install TestFloat"
+          working_directory: /tmp
+          command: |
+            if [ ! -f testfloat_gen ]; then
+              sudo apt-get -q update
+              sudo apt-get -qy install --no-install-recommends unzip
```

Comment:
Could have added unzip to cpp-build-env 😝

Reply:
Fixed with github.